pagecontent/schema/pagecontent.xsd

```diff
@@ -116,6 +116,11 @@
 		<attribute name="imageFilename" type="string" use="required"></attribute>
 		<attribute name="imageWidth" type="int" use="required"></attribute>
 		<attribute name="imageHeight" type="int" use="required"></attribute>
+		<attribute name="imageXResolution" type="int" use="required"></attribute>
```

Comment:
I do not think that `required` is a good idea! Consider e.g. backward compatibility!!!